CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 ## 5.12.0 (Unreleased)
 
+FEATURES:
+* **New Data Source:** `google_dns_managed_zones` ([#6835](https://github.com/hashicorp/terraform-provider-google-beta/pull/6835))
+* **New Data Source:** `google_filestore_instance` ([#6822](https://github.com/hashicorp/terraform-provider-google-beta/pull/6822))
+* **New Data Source:** `google_vmwareengine_external_access_rule` ([#6811](https://github.com/hashicorp/terraform-provider-google-beta/pull/6811))
+* **New Resource:** `google_clouddomains_registration` ([#6833](https://github.com/hashicorp/terraform-provider-google-beta/pull/6833))
+* **New Resource:** `google_netapp_kmsconfig` ([#6831](https://github.com/hashicorp/terraform-provider-google-beta/pull/6831))
+* **New Resource:** `google_vertex_ai_feature_online_store_featureview` ([#6821](https://github.com/hashicorp/terraform-provider-google-beta/pull/6821))
+* **New Resource:** `google_vmwareengine_external_access_rule` ([#6811](https://github.com/hashicorp/terraform-provider-google-beta/pull/6811))
+
+IMPROVEMENTS:
+* compute: added `md5_authentication_key` field to `google_compute_router_peer` resource ([#6815](https://github.com/hashicorp/terraform-provider-google-beta/pull/6815))
+* compute: added in-place update support to `params.resource_manager_tags` field in `google_compute_instance` resource ([#6828](https://github.com/hashicorp/terraform-provider-google-beta/pull/6828))
+* compute: added in-place update support to `description` field in `google_compute_instance` resource ([#6804](https://github.com/hashicorp/terraform-provider-google-beta/pull/6804))
+* gkehub: added `policycontroller` field to `google_gke_hub_feature_membership` resource. Users can now configure a Policy Controller for individual memberships in the fleet. ([#6813](https://github.com/hashicorp/terraform-provider-google-beta/pull/6813))
```

Comment:
`Users can now configure a Policy Controller for individual memberships in the fleet`

Not sure what this part means. If it's just a reiteration that the field has been added we can likely remove it.

Reply:
It is just a reiteration. Removed.